Sources/ScipioKit/Producer/PIF/ModuleMapGenerator.swift

```diff
@@ -56,36 +56,45 @@ struct ModuleMapGenerator {
                 return try convertCustomModuleMapForFramework(customModuleMap)
                     .trimmingCharacters(in: .whitespacesAndNewlines)
             case .umbrellaHeader(let headerPath):
-                return """
-                framework module \(context.resolvedTarget.c99name) {
-                    umbrella header "\(headerPath.basename)"
-                    export *
-                }
-                """
-                    .trimmingCharacters(in: .whitespaces)
+                return ([
+                    "framework module \(context.resolvedTarget.c99name) {",
+                    "    umbrella header \"\(headerPath.basename)\"",
+                    "    export *",
+                ]
+                + generateLinkSection(context: context)
```

Comment:
Add `link framework` declares on generated modulemap to be embedded in frameworks.

```
framework module MyModule {
    umbrella header "MyModule.h"
    export *
    link framework "MyDependency1"
    link framework "MyDependency2"
}
```

https://clang.llvm.org/docs/Modules.html#link-declaration